src/main/java/ru/mercuriev/game/of/life/NeighboursCounter.java

```diff
@@ -0,0 +1,36 @@
+package ru.mercuriev.game.of.life;
+
+/**
+ * @author paul
+ */
+class NeighboursCounter {
+
+    private int[][] world;
+
+    public static NeighboursCounter getInstance(int[][] world) {
```

Comment:
As far as i can understnd, Effective Java by J. Bloh (2nd edition page 10) says we have two options: **newInstance** or **of**, I vote for the first